ruby/README.md

```diff
@@ -59,10 +59,20 @@ See full example of [request signing].
 
 ## Verifying webhooks
 
-The `verify_with_pem` method may be used to verify webhook `Tl-Signature` header signatures.
+The `verify_with_jwks` method may be used to verify webhook `Tl-Signature` header signatures.
 
 ```ruby
-TrueLayerSigning.verify_with_pem(pem)
+# The `jku` field is included in webhook signatures
+jku = TrueLayerSigning.extract_jws_header(webhook_signature).jku
+
+# You should check that the `jku` is a valid TrueLayer URL (not provided by this library)
+ensure_jku_allowed(jku)
```

Comment:
Is there a reason for not doing this as part of the library?

Reply:
Oh I see, the SDKs do that. Ignore me.

Reply:
I'm not sure why we decided not to build this into the libraries, however we do provide an example of how it can be achieved.